### DISCUSSION TITLE
Nuxt is deployed on a Linux server

### DISCUSSION THREAD

**Discussion by zk-qd** [nuxt/nuxt#9838] [Questions (Nuxt 2)]
![image](https://user-images.githubusercontent.com/49986675/134448721-aaa9f901-a258-4be9-bd7f-ee755903aa60.png)
The "nuxt" local build project is fine, but building a project in "Linux" will prompt you that you can't find the image

**Comment by kissu** [comment]
What is the `nuxt local build` even ? Can you share your `nuxt.config.js` please? 😁
